aws/rapids_ec2_mnmg.ipynb

```diff
@@ -0,0 +1,507 @@
+{
+ "cells": [
```

Comment:
What about the local env that they're running this notebook?

Reply:
Added instructions in README